docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,376 @@
+[//]: # (title: Migrate to Dokka Gradle plugin v2)
+
+> The Dokka Gradle plugin v2 is an [experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained) feature. 
+> It may be changed at any time. We appreciate your feedback on [GitHub](https://github.com/Kotlin/dokka/issues).
+>
+{type="warning"}
+
+The Dokka Gradle plugin (DGP) is a tool to generate comprehensive API documentation for Kotlin projects.
```

Comment:
Something that caught my eye, and I'm curious about. Normally the original term of an abbreviation is capitalized. Should 'plugin' be capitalised? I would expect it to be "Dokka Gradle Plugin (DGP)". Or is this a stylistic choice?

Reply:
Yes, it's a stylistic choice. In our docs, when describing plugins, we don't consider "plugin" as part of the proper name, so it stays lowercase. However, "P" can be part of the abbreviation. This is consistent with other cases like [KGP](https://kotlinlang.org/docs/gradle-plugin-variants.html#gradle-plugin-variants.md) = Kotlin Gradle plugin